rotkehlchen/tests/api/test_current_assets_price.py

```diff
@@ -527,3 +531,78 @@ def test_prices_cache_invalidation_for_manual_prices(rotkehlchen_api_server):
     # # the max diff is to account for price changes between the requests interval
     assert FVal(avax_result['assets']['AVAX'][0]).is_close(FVal(eth_result['assets']['ETH'][0]), max_diff='10')  # noqa: E501
     assert result['target_asset'] == 'USD'
+
+
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
+@pytest.mark.parametrize('ethereum_accounts', [['0xc37b40ABdB939635068d3c5f13E7faF686F03B65']])
+@pytest.mark.parametrize('ethereum_modules', [['nfts']])
+def test_get_manual_prices_with_nfts(
+    rotkehlchen_api_server: 'APIServer',
```

Comment:
shouldnt' this need 1 more indentation level?

Reply:
I have now the correct indentation in the editor but might have edited it myself